packages/mc-scripts/src/vite-plugins/vite-plugin-svgr.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * COPIED FROM https://github.com/pd4d10/vite-plugin-svgr
```

Comment:
Just curious about why we copied the file instead of importing the dependency into the project. Is it because we don't want to maintain another dependency when it only contains one file? would we need to manually check for new versions?

Reply:
Yes so mostly I wanted to make sure the plugin implementation was compatible with Vite 3. Since the implementation is pretty simple, and we only need specific functionality of it, I thought to just copy it over and remove some of the unused parts.

We can always switch back to using the package if we want to.